indico/web/client/js/exports.js

```diff
@@ -46,5 +47,6 @@ window._IndicoReactI18n = IndicoReactI18n;
 window._IndicoReactUtil = IndicoReactUtil;
 window._IndicoUtilsAxios = IndicoUtilsAxios;
 window._IndicoUtilsDate = IndicoUtilsDate;
+window._IndicoUtilsCase = IndicoUtilsCase;
```

Comment:
Why is this export needed? The code in that module looks simple enough to work fine without exporting it (importing JS without an export like the ones here just means the code gets duplicated in the plugin's JS bundle)

Reply:
We usually get an error when trying to import something that has not been exported, for this as well. Example error:
```
Uncaught TypeError: __webpack_require__.t is not a function
    at ./js/exports.js (exports.js:32:27)
    at __webpack_require__ (bootstrap:19:1)
    at __webpack_exec__ (exports.js:52:64)
    at exports.js:52:64
    at __webpack_require__.O (chunk loaded:23:1)
    at webpackJsonpCallback (jsonp chunk loading:44:1)
    at Array.forEach (<anonymous>)
    at jsonp chunk loading:48:1
    at jsonp chunk loading:49:1
    at startup:5:1
```

```
isArrayLikeObject.js:33 Uncaught TypeError: __webpack_require__.nmd is not a function
    at ../../../node_modules/lodash/isBuffer.js (isArrayLikeObject.js:33:1)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
    at ../../../node_modules/lodash/isEmpty.js (isEmpty.js:6:16)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
    at ../../../node_modules/flask-urls/lib/build.js (build.js:14:39)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
    at ./js/react/components/principals/PrincipalField.jsx (PermissionTree.jsx:88:42)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
    at ./js/react/components/principals/index.js (imperative.js:61:1)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
```